### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by jGleitz**
Made minor improvements of grammar and wording.

Moved the record to  "`meeting-minutes`" in "`doc`".

### CODE REVIEW

**Comment by RomanLangrehr** [comment]
I dont think:
"For the implementation phase there are two leaders needed"
is correct english.
What about:
"We need two leaders for the implementation phase"
?

File: doc/meeting-minutes/meeting_2015_11_13.md [hunk 1]
```diff
@@ -18,4 +18,4 @@
 
 - Read chapters 1,2,3,5 of Klaus Krogmann's [thesis](https://sdqweb.ipd.kit.edu/publications/pdfs/krogmann2010a.pdf)
 
-- Everyone should think about whether he/she prefers a several development phase to lead (For the implementation phase there are two leaders needed).
+- Everyone should think about whether he/she prefers a specific development phase to manage (There will be two managers eneded for the implementation phase).
```
Comment (RomanLangrehr) [inline doc/meeting-minutes/meeting_2015_11_13.md]: I dont think:

> "For the implementation phase there are two leaders needed"

is correct english.
What about:
"We need two leaders for the implementation phase"
?

Comment (jGleitz) [inline doc/meeting-minutes/meeting_2015_11_13.md]: That's why I wrote 

> There will be two managers needed for the implementation phase

**Comment by jGleitz** [comment]
I accidentally merged Beagle-PSE/Beagle#4 and therefore opened Beagle-PSE/Beagle#5. Please contunue discussion there.

**Pull request closed without merging**